course/src/main/java/in/testpress/course/ui/AvailableCourseListFragment.java

```diff
@@ -43,6 +66,62 @@ public void onCreate(Bundle savedInstanceState) {
         courseDao = TestpressSDKDatabase.getCourseDao(getActivity());
         productDao = TestpressSDKDatabase.getProductDao(getContext());
         pager = new CourseProductPager(apiClient);
+        productCategoriesViewModel = new ViewModelProvider(
+                this,
+                new ProductCategoriesViewModelFactory(
+                        new ProductCategoriesRepository(requireContext())
+                )
+        ).get(ProductCategoriesViewModel.class);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.available_course_fragment, container, false);
+    }
+
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        bind(view);
+        initProductCategoriesView();
+        initViewModel();
+    }
+
+    private void bind(View view){
+        productCategoriesLayout = (CardView) view.findViewById(R.id.product_categories_layout);
+        productCategoriesRecyclerView = (RecyclerView) view.findViewById(R.id.product_categories_list);
+    }
+
+    private void initProductCategoriesView(){
+        productCategoriesAdapter = new ProductCategoriesAdapter(requireContext(),this);
+        productCategoriesRecyclerView.setLayoutManager(new LinearLayoutManager(requireContext(),LinearLayoutManager.HORIZONTAL,false));
+        productCategoriesRecyclerView.setAdapter(productCategoriesAdapter);
+    }
+
+    private void initViewModel(){
+        initalizeObservers();
+        productCategoriesViewModel.loadCategories();
```

Comment:
This name is not clear. I am not getting what load means. 
Load can mean loading from local or getting from internet. Appropriate name should be `fetchCategories`